README.md

```diff
@@ -274,13 +266,30 @@ You can use the following syntax for adding callouts, but prefer the simpler mar
 ```
 
 ### How can I add/fix images in an article?
+
 1. Access the [dev-portal-content/docs/guides](https://github.com/vtexdocs/dev-portal-content/tree/main/docs/guides) folder.
 2. Upload the images you wish to include in your guide in the same folder your article is located.
 3. Access the file in Github web.
 4. Open the raw version of the uploaded file by opening the image in a new tab.
 5. Save the URL.
 6. Mention the URL in your markdown.
 
+### How can I hide articles from search engines?
+
+When you publish an article in Developer Portal and wish only those with the link can access it, you can hide it from searching engines. To do so, when creating the article, fill in the header with the value `hidden: true`.
```

Comment:
Acrescentaria um use case tipo "this is useful for products in Beta that need documentation for specific clients, instead of publicly available docs".